CourseApp/Employee.cs

```diff
@@ -2,29 +2,41 @@
 
 namespace CourseApp
 {
-    public class Employee
+    public class Employee : Person
     {
-        private int age;
         private int workDays;
 
         public Employee()
-        : this(14)
+        : base()
         {
+            DateOfEmploy = DateTime.Now;

```

Comment:
lets create private method with initializing default parameters and call it from constructor